### PR TITLE
LPS-151632 Move petra-portlet-url-builder to apps/portal

### PR DESCRIPTION
**Pull request by julschong**

### CODE REVIEW

**Comment by kyle-miho** [comment]
ci:test:sf

**Comment by kyle-miho** [comment]
Other failures should be unrelated

**Comment by julschong** [comment]
@kyle-miho Hi Kyle,
SF is an unrelated typo. last commit should fix it

**Comment by julschong** [comment]
ci:test:sf

File: modules/core/portal-bootstrap/system.packages.extra.bnd [hunk 1]
```diff
@@ -290,12 +290,12 @@ Provide-Capability:\
...
-	@lib/com.liferay.petra.portlet.url.builder.jar,\
 	@lib/com.liferay.petra.process.jar,\
 	@lib/com.liferay.petra.reflect.jar,\
 	@lib/com.liferay.petra.sql.dsl.api.jar,\
 	@lib/com.liferay.petra.sql.dsl.spi.jar,\
 	@lib/com.liferay.petra.string.jar,\
 	@lib/com.liferay.petra.url.pattern.mapper.jar,\
-	@lib/com.liferay.petra.xml.jar
+	@lib/com.liferay.petra.xml.jar,\
+	@lib/com.liferay.portal.portlet.url.builder.jar
```
Comment (tinatian) [inline modules/core/portal-bootstrap/system.packages.extra.bnd:300]: @julschong No need to do this, remove it directly.

Tina

**Pull request closed without merging**